### ISSUE TITLE
Feature: Weekly Email Summaries

### ISSUE THREAD

**Issue by mattttt** [raintank/worldping-app#161]
Trying to think about positive ways to stay front-on-mind for the free users. Currently, if you're under 3M checks, you could easily forget you even have an account, and never be bothered. 

Instead of marketing to these folks, I think a better approach would be to create some value in a weekly summary of endpoints.

Not sure how we'd do this, but I think there are some basic stats that we could pull out for endpoints and present in a visually appealing way via email.

Your Weekly worldPing Summary:
Endpoint Uptime
Problem Areas (count of errors by probe?)
Alerts this past week (way to remind people that they could enable alerting if they hadnt)

There is likely more we can do here. Will continue to give it thought.

@woodsaj: How feasible is this? How much effort?

**Comment by woodsaj** [comment]
This would be at least a weeks worth of work, So given everything else that needs to be done, i wouldnt expect anything before the end of the year.

**Comment by mattttt** [comment]
Yep, not quite interested in plowing ahead without discussion. Just wanted to understand the size of the request. Thanks!

**Comment by rcornett** [comment]
This is something that would add great value for my company. Currently we use another provider that sends out weekly email summaries which the executive team rely on. If this is something you could add in it would be greatly appreciated.
